src/async/campaigns.js

```diff
@@ -43,6 +43,33 @@ export async function startCampaign(name, congId, publisherId) {
   return results.insertId;
 }
 
+export async function updateCampaign(campaignId, name, start_date, end_date) {
+
+  const campaignToUpdate = await pool.query(`Select name from campaigns WHERE id = ${campaignId}`);
```

Comment:
we can just let the UPDATE... WHERE condition do the check if the campaign id exists. We don't need to check first.

Reply:
done